### PR TITLE
fix: pass resolved `runtimeConfig` into nuxt environment

### PR DESCRIPTION
**Pull request by floriankapaun**
Resolves danielroe/nuxt-vitest#29

### CODE REVIEW

File: packages/nuxt-vitest/src/config.ts [hunk 1]
```diff
@@ -61,6 +61,9 @@ export async function getVitestConfigFromNuxt(
     test: {
       ...options.viteConfig.test,
       dir: options.nuxt.options.rootDir,
+      environmentOptions: {
+        nuxtRuntimeConfig: options.nuxt.options.runtimeConfig,
```
Comment (danielroe) [inline packages/nuxt-vitest/src/config.ts:65]:
```suggestion
        ...options.viteConfig.test.environmentOptions,
        nuxtRuntimeConfig: options.nuxt.options.runtimeConfig,
```

File: playground/tests/nuxt/config.spec.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { it, expect } from 'vitest'
+
+it('should return the runtimeConfig from nuxt.config', () => {
+    const config = useRuntimeConfig();
+    expect(config).toBeTypeOf('object');
+    expect(config?.public).toEqual({
+        hello: 'world'
+    })
+})
```
Comment (danielroe) [inline playground/tests/nuxt/config.spec.ts:9]:
```suggestion
  const config = useRuntimeConfig();
  expect(config).toBeTypeOf('object');
  expect(config?.public).toEqual({
    hello: 'world'
  })
})
```

**Review by danielroe — APPROVED** [review]
This looks beautiful. Thank you ❤️

**Comment by hecktarzuli** [comment]
It would appear the env overrides like NUXT_PUBLIC_FOO=true don't come through.

**Pull request merged**